nums/core/array/application.py

```diff
@@ -1116,3 +1116,86 @@ def nanmean(self, a: BlockArray, axis=None, keepdims=False, dtype=None):
         if dtype is not None:
             res = res.astype(dtype)
         return res
+
+    def nanvar(self, a: BlockArray, axis=None, ddof=0, keepdims=False, dtype=None):
```

Comment:
Do we have test coverage for nanvar and nanstd?

Reply:
Yes, in `test_nan_reductions`.